zim/plugins/tableofcontents.py

```diff
@@ -396,8 +424,30 @@ def on_populate_popup(self, treeview, menu):
 			else:
 				item.set_sensitive(False)
 
+		sep = Gtk.SeparatorMenuItem()
+		menu.append(sep)
+
+		item = Gtk.MenuItem.new_with_mnemonic(_("Copy _link to this location"))
```

Comment:
Minor: for translation strings, please add a comment like "# T: purpose of string" this is picked up as info for translators